### ISSUE TITLE
Gratipay link is broken

### ISSUE THREAD

**Issue by frinyvonnick** [vim-dist/webvim#62]
Hi,

The Gratipay link in the README.md is broken 😢

**Comment by krampstudio** [comment]
Yes, it looks like the service has been replaced by LibrePay. I'll update it .
Thank you for the feedback!
